diskwriter_test.go

```diff
@@ -158,17 +158,16 @@ func TestWalkerWriterSimple(t *testing.T) {
 	err = Walk(context.Background(), dest, nil, bufWalk(b))
 	assert.NoError(t, err)
 
-	assert.Equal(t, string(b.Bytes()), `dir bar
+	assert.Equal(t, `dir bar
```

Comment:
Swapped the arguments to match the error message better - expected is the first followed by actual. Otherwise, the error message on failure is confusing.